Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:8-slim
+
+WORKDIR /usr/src/app
+
+# Keep the .git directory in order to properly report version
+COPY ./package.json .
+
+RUN apt-get update \
+    && apt-get install -y jq python git build-essential vim --no-install-recommends \
+    && npm install
+
+COPY ./ ./
+
+ENV NO_PROXY localhost,127.0.0.1
+ENV no_proxy localhost,127.0.0.1
+
+ENTRYPOINT ["/usr/src/app/docker-entrypoint.sh"]
```

Comment:
I think it would be best to remove the `ENTRYPOINT` line since there is no entry point script. It causes errors if you try to run the docker image standalone.